src/index.ts

```diff
@@ -445,14 +445,14 @@ function readConfig (compilerOptions: any, project: string | boolean | undefined
   delete result.config.compilerOptions.outFile
   delete result.config.compilerOptions.declarationDir
 
-  const basePath = result.path ? dirname(result.path) : cwd
+  const basePath = result.path ? normalizeSlashes(dirname(result.path)) : cwd
```

Comment:
It looks like you should be normalizing the slashes on both sides, not just the left side of the ternary.